### PR TITLE
Created first iteration of PR template

### PR DESCRIPTION
**Pull request by lyndsiWilliams**
## Summary
This is the first iteration of our pull request template!

## Testing instructions
- Navigate to `GG-JointJustice-Unity/.github/pull_request_template.md`
- Observe the new PR template file

## Additional information

- `[ ]` Changes UI
- [x] Introduces new feature
- `[ ]` Removes existing feature
- `[ ]` Has associated resource:

### CODE REVIEW

**Review by notexplosive — APPROVED** [review]
I'm good with this! Couple nits that are optional, you have my approval either way :)

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+### SUMMARY
```
Comment (notexplosive) [inline .github/pull_request_template.md:1]: I don't love the SHOUTY CAPITALS but I also don't care.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+<!--- Describe the change below, including rationale and design decisions -->
+
+### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
```
Comment (notexplosive) [inline .github/pull_request_template.md:4]: nit: consider ## (h2) instead of ### (h3). We're using h2 in the other template, not a huge deal either way

**Review by ghost — APPROVED** [review]

**Comment by notexplosive** [comment]
I have another tiny gripe: I like the idea of checkboxes indicating what "type" of change it is, however it looks like GitHub thinks those unchecked boxes are "unfinished tasks." It sees unchecked boxes and wants them to be checked. Maybe instead of checkboxes we do `[ ]` and `[x]` wrapped in backticks?

image

File: .github/pull_request_template.md (hunk 1 above)
Comment (lyndsiWilliams) [reply .github/pull_request_template.md]: Good point, made things less anxiety-inducing in [`this commit`](https://github.com/Studio-Lovelies/GG-JointJustice-Unity/pull/7/commits/228fd318abdc7cf9d992678606958d7ad9700017) 😁

File: .github/pull_request_template.md (hunk 2 above)
Comment (lyndsiWilliams) [reply .github/pull_request_template.md]: You got it! Made things smol on [`this commit`](https://github.com/Studio-Lovelies/GG-JointJustice-Unity/pull/7/commits/228fd318abdc7cf9d992678606958d7ad9700017) 👶

**Comment by notexplosive** [comment]
You've got two approvals, merge when ready :)

**Review by JuiceGrape — APPROVED** [review]

**Pull request merged**